config/webpack.config.js

```diff
@@ -85,10 +85,16 @@ var config = {
       },
       {
         test: /\.scss$/,
-        use: ExtractTextPlugin.extract({
-          fallback: 'style-loader', // The backup style loader
-          use: 'css-loader?sourceMap!sass-loader?sourceMap'
-        })
+        use: [
+          'style-loader',
+          'css-loader',
+          {
+            loader: 'sass-loader',
+            options: {
+              includePaths: [path.resolve(__dirname, '..', 'node_modules')]
```

Comment:
Perhaps there is a better way to do this, let me know if you know it.

Reply:
@waldenraines isn't the `ExtractTextPlugin.extract` necessary for the `ExtractTextPlugin` to perform well?

Reply:
Oops, didn't mean to remove that but when I use it I get a bunch of these:

```
17:12:24 webpack.1 | ERROR in ./webpack/assets/javascripts/react_app/components/common/charts/PieChart/PieChart.scss
17:12:24 webpack.1 | Module build failed: ReferenceError: self is not defined
17:12:24 webpack.1 |     at /home/vagrant/foreman/node_modules/style-loader/index.js!/home/vagrant/foreman/node_modules/css-loader/index.js!/home/vagrant/foreman/node_modules/sass-loader/lib/loader.js??ref--3-4!/home/vagrant/foreman/webpack/assets/javascripts/react_app/components/common/charts/PieChart/PieChart.scss:801:30
17:12:24 webpack.1 |     at /home/vagrant/foreman/node_modules/style-loader/index.js!/home/vagrant/foreman/node_modules/css-loader/index.js!/home/vagrant/foreman/node_modules/sass-loader/lib/loader.js??ref--3-4!/home/vagrant/foreman/webpack/assets/javascripts/react_app/components/common/charts/PieChart/PieChart.scss:796:47
17:12:24 webpack.1 |     at module.exports.module.exports (/home/vagrant/foreman/node_modules/style-loader/index.js!/home/vagrant/foreman/node_modules/css-loader/index.js!/home/vagrant/foreman/node_modules/sass-loader/lib/loader.js??ref--3-4!/home/vagrant/foreman/webpack/assets/javascripts/react_app/components/common/charts/PieChart/PieChart.scss:818:68)
17:12:24 webpack.1 |     at Object.module.exports.module.exports.list (/home/vagrant/foreman/node_modules/style-loader/index.js!/home/vagrant/foreman/node_modules/css-loader/index.js!/home/vagrant/foreman/node_modules/sass-loader/lib/loader.js??ref--3-4!/home/vagrant/foreman/webpack/assets/javascripts/react_app/components/common/charts/PieChart/PieChart.scss:712:36)
17:12:24 webpack.1 |     at __webpack_require__ (/home/vagrant/foreman/node_modules/style-loader/index.js!/home/vagrant/foreman/node_modules/css-loader/index.js!/home/vagrant/foreman/node_modules/sass-loader/lib/loader.js??ref--3-4!/home/vagrant/foreman/webpack/assets/javascripts/react_app/components/common/charts/PieChart/PieChart.scss:639:30)
17:12:24 webpack.1 |     at /home/vagrant/foreman/node_modules/style-loader/index.js!/home/vagrant/foreman/node_modules/css-loader/index.js!/home/vagrant/foreman/node_modules/sass-loader/lib/loader.js??ref--3-4!/home/vagrant/foreman/webpack/assets/javascripts/react_app/components/common/charts/PieChart/PieChart.scss:685:37
17:12:24 webpack.1 |     at Object.<anonymous> (/home/vagrant/foreman/node_modules/style-loader/index.js!/home/vagrant/foreman/node_modules/css-loader/index.js!/home/vagrant/foreman/node_modules/sass-loader/lib/loader.js??ref--3-4!/home/vagrant/foreman/webpack/assets/javascripts/react_app/components/common/charts/PieChart/PieChart.scss:688:10)
17:12:24 webpack.1 |     at Module._compile (module.js:570:32)
17:12:24 webpack.1 |     at Object.exec (/home/vagrant/foreman/node_modules/webpack/lib/NormalModule.js:129:12)
17:12:24 webpack.1 |     at /home/vagrant/foreman/node_modules/extract-text-webpack-plugin/dist/loader.js:131:26
17:12:24 webpack.1 |  @ ./webpack/assets/javascripts/react_app/components/common/charts/PieChart/PieChart.scss
17:12:24 webpack.1 |  @ ./webpack/assets/javascripts/react_app/components/common/charts/PieChart/index.js
17:12:24 webpack.1 |  @ ./webpack/assets/javascripts/react_app/components/componentRegistry.js
17:12:24 webpack.1 |  @ ../katello/webpack/index.js
17:12:24 webpack.1 |  @ multi (webpack)-dev-server/client?http://0.0.0.0:3808 webpack/hot/dev-server ../katello/webpack/index.js
17:12:24 webpack.1 | 
```

Any ideas?  I am no expert on webpack.

Reply:
Never mind I think I got it.  Updated.